.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] Please Make sure that the last file(test.md file) is actually ignored by git and is not present in your GitHub repository. you can use 

`# ignore Test.md file`
 ` test.md`
Still I Can see it in your Repo: 
![c](https://user-images.githubusercontent.com/93598189/185660173-a0b4f4c2-9769-4b8d-804a-01920165615c.PNG)